File: server/src/main/java/ru/fullrest/mfr/server/ServerApplication.java
package ru.fullrest.mfr.server;

import lombok.RequiredArgsConstructor;
import org.springframework.boot.CommandLineRunner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
import ru.fullrest.mfr.server.telegram.TelegramBot;

@SpringBootApplication
@RequiredArgsConstructor
@EnableJpaRepositories
public class ServerApplication implements CommandLineRunner {

    private final TelegramBot telegramBot;

    public static void main(String[] args) {
        SpringApplication.run(ServerApplication.class, args);
    }

    @Override
    public void run(String... args) throws Exception {
        telegramBot.setWebhook(telegramBot.getBotPath() + telegramBot.getBotToken(), null);
    }
}

File: server/src/main/kotlin/com/lezenford/mfr/server/ServerApplication.kt
package com.lezenford.mfr.server

import com.lezenford.mfr.server.configuration.properties.ServerSettingProperties
import com.lezenford.mfr.server.configuration.properties.TelegramProperties
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableConfigurationProperties(value = [ServerSettingProperties::class, TelegramProperties::class])
class ServerApplication

fun main(args: Array<String>) {
    runApplication<ServerApplication>(*args)
}